iowrappers/users.go

```diff
@@ -34,10 +34,11 @@ type GoogleOAuthResponse struct {
 }
 
 type PlanningEvent struct {
-	User      string `json:"user"`
-	City      string `json:"city"`
-	Country   string `json:"country"`
-	Timestamp string `json:"timestamp"`
+	User              string `json:"user"`
+	City              string `json:"city"`
+	Country           string `json:"country"`
+	AdminAreaLevelOne string `json:"admin_area_level_one"`
```

Comment:
what does this new Variable do? AdminAreaLevelOne

Reply:
`AdminAreaLevelOne` is a generic term for a country's largest subnational administrative unit. For example, California